webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/FmcScrambleCutoutSheet.kt

```diff
@@ -71,16 +72,15 @@ class FmcScrambleCutoutSheet(request: ScrambleRequest, globalTitle: String?): Fm
     }
 
     companion object {
-        val BASE_FONT = FontUtil.getFontForLocale(Translate.DEFAULT_LOCALE)
+        val baseFont = FontUtil.getFontForLocale(Translate.DEFAULT_LOCALE)
+        val font = Font(baseFont, 20f)
```

Comment:
1. constant values in (companion) objects are written in SNAKE_CASE by convention, because they effectively act as constants.
2. `font` objects are bound to their particular PDF in itext5. I don't see why creating a font out of the baseFont object here is beneficial...